### PR TITLE
Save the commit version to a file to add a commit before we push tags

### PR DESCRIPTION
**Pull request by praksb**
### What does this PR do?
Adds a commit before we push tags.

### CODE REVIEW

File: scripts/publish.js [hunk 1]
```diff
@@ -81,6 +81,9 @@ if (/master/.test(currentBranch)) {
 
 // Create a git tag if we are publishing a specific version
 if (!/latest/.test(DOCKER_IMAGE_VERSION)) {
+  shell.exec(`echo ${DOCKER_IMAGE_VERSION} > version.txt`);
```
Comment (vazexqi) [inline scripts/publish.js]: Do you want to add version.txt to the .dockerignore? Or do you want that included as part of the docker image?

File: scripts/publish.js [hunk 2]
```diff
@@ -81,6 +81,9 @@ if (/master/.test(currentBranch)) {
...
+  shell.exec(`git add .`);
```
Comment (vazexqi) [inline scripts/publish.js]: Might be safer to just add the file that you modified. `git add .` might add more and surprise us especially if there are other changes in the folder that were accidentally added.

Comment (praksb) [reply scripts/publish.js]: Good point. let me fix that.

File: scripts/publish.js (hunk 1 above)
Comment (praksb) [reply scripts/publish.js]: let me add it to .dockerignore for now.

**Pull request merged**